app/infrastructure/database/orm/job_skill_orm.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# http://sequel.jeremyevans.net/rdoc-plugins/classes/Sequel/Plugins/Timestamps.html
```

Comment:
Please remove this commented-out code.

Reply:
Issue solved. Please refer to this line of the [commit](https://github.com/KeroKnows/s-killer/commit/c1d43b9257897b20edca40169e25f234a0879b85#diff-cdc2ae7dba7afb211ecd7c0f113e19ae424730b47e9d212d9b5cf109fd8cce6eL3).